README.md

```diff
@@ -95,20 +95,82 @@ If you're not already running a stats collection service, you should take a look
 Most people will only need to specify [the config](config/default.yaml) they're interested in
 and start up the server.
 
-If you need more customization, you can write a module:
+Configuration Options:
+
+- `server:` {Object}  
+Use to set properties of the Bucky Server.
+  - `port:` {Number}  
+  Use to set the port that Bucky Server will listen to.
+  - `appRoot:` {String}  
+  Use to define the root of the endpoint.
+  - `https:` {Object}  
+  Defines a set of options for running Bucky in https mode.
+    - `port:` {Number}  
+    Use to specify the port for https, if not populated the default is the http server port + 1.
+    - `options:` {Object}  
+    Use to define the options for https.
+    key and cert are mandatory options, here is a full [list of all available options](https://nodejs.org/api/tls.html#tls_tls_createserver_options_secureconnectionlistener).
+    For all options that accept a buffer you can use the path to the file containing the option's data.
+     - `key:` {Object|String|Buffer}  
```

Comment:
in the example config below key is a sub-object of server.https.options. Here it is directly under server.https. Which is the right place in the hierarchy for it? Same question goes for cert.